src/System.Globalization.Calendars/tests/GregorianCalendar/GregorianCalendarTests.Utilities.cs

```diff
@@ -6,7 +6,21 @@ namespace System.Globalization.Tests
 {
     public class GregorianCalendarTestUtilities
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
+        [ThreadStatic]
+        private static RandomDataGenerator s_randomDataGenerator;
+
+        private static RandomDataGenerator RDG
```

Comment:
I know this is just a test but can we call it something nicer, e.g. RandomDataGenerator?